src/parameterizations/lateral/MOM_lateral_mixing_coeffs.F90

```diff
@@ -611,11 +618,13 @@ end subroutine calc_sqg_struct
 
 !> Calculates and stores functions of isopycnal slopes, e.g. Sx, Sy, S*N, mostly used in the Visbeck et al.
 !! style scaling of diffusivity
-subroutine calc_slope_functions(h, tv, dt, G, GV, US, CS, OBC)
+subroutine calc_slope_functions(h, uh, vh, tv, dt, G, GV, US, CS, OBC)
   type(ocean_grid_type),                     intent(inout) :: G  !< Ocean grid structure
   type(verticalGrid_type),                   intent(in)    :: GV !< Vertical grid structure
   type(unit_scale_type),                     intent(in)    :: US !< A dimensional unit scaling type
   real, dimension(SZI_(G),SZJ_(G),SZK_(GV)), intent(inout) :: h  !< Layer thickness [H ~> m or kg m-2]
+  real, dimension(SZIB_(G),SZJ_(G),SZK_(GV)),intent(inout) :: uh !< Layer thickness times u [UH ~> m2 s-1 or kg m-1 s-1]
```

Comment:
The units here are described incorrectly, and the variables that are being used in calls to `calc_slope_functions()` are not just the velocities times thicknesses but rather they are the total thickness transports through the faces in units of `[H L2 T-1 ~> m3 s-1 or kg s-1]`.